app.js

```diff
@@ -0,0 +1,39 @@
+//#region SERVER SETUP
+
+const { exec } = require('child_process');
+const express = require('express'),
+    app = express(),
+    port = 3000;
+
+// Express Middleware for serving static files.
+app.use(express.static(__dirname + '/public'));
+
+//Kicks off the app on a specified port
+app.listen(port);
+console.log(`Your server available at http://localhost:${port}`);
+
+//#endregion
+
+//#region ROUTES
+
+//The landing page of the site. 
+app.get('/', function (req, res) {
+    res.redirect('index.html');
+});
+
+//Endpoint to load the names of the birds. 
+app.get('/birdnames', (req, res) => {
```

Comment:
Endpoints don't have to return html though, and you can use them to load data on the front end. If you try going to localhost:3000/birdnames, you'll see the json data that I am returning from the server 
![Capture](https://github.com/pjbabcock/birds-at-my-feeder/assets/98671484/4730e497-e393-4701-8829-a342ed4c2365)
for the autocomplete.